### DISCUSSION TITLE
Discogs Plugin removing the feat. from the Trackname

### DISCUSSION THREAD

**Discussion by silkties** [beetbox/beets#4873] [Q&A]
Hey Guys, 
I've started using beets just recently and after taking a lot of time to write my config i've stumbled across one problem.
When autotagging i prefer discogs over Musicbrainz, the problem here is, that when i do so, it doesnt write any genre tags, as well as, and thats more of a problem for me, it doesnt fetch and write any featuring artists, not in the artist tag or the track tag. 

Is there actually any possibility to do that or am i just not seeing the obvious? 
Thanks in advance for any help!

**Comment by JOJ0** [comment]
Hmm not sure if it works but genre tag should be written.

https://beets.readthedocs.io/en/latest/plugins/discogs.html#configuration

Also try append_style_genre

For the feat. thing try this plugin: https://beets.readthedocs.io/en/latest/plugins/ftintitle.html

But if they are missing in artist already, that wont help probably...

Hmmm just some random hints, hope some of it helps moving closer to a solution or even finding a bug :-)

**Reply by JOJ0** [reply]
@silkties did that help?

**Reply by silkties** [reply]
partially! the append_style_genre plugin helped, the second issue didnt really got resolved. the problem is, discogs adds the featuring artists in a extra column, therefore one would have to implement fetching that columns and adding it to the title via ftintitle. in any case, much thx for the support!

**Reply by JOJ0** [reply]
So the 2nd issue is that discogs doesnt write the feat. artist into the artist field nor into the title field? correct?

maybe you could wrap all this including examples of what works and what doesnt into an issue/bug report?

that would help and let's hope somebody picks it up. I have some knowledge of the discogs plugin and can support/review.

thanks in advance!

**Reply by JOJ0** [reply]
Ah, and just to make sure: We are talking about the latest beets version on master branch and not 1.6.0?

I recall I fixed something related to feat. artists a couple of months ago but need to investigate what it was exactly ;-)

Update: I think I fixed this in https://github.com/beetbox/beets/pull/4515

Does it work if you install latest beets version like so: https://beets.readthedocs.io/en/latest/contributing.html#getting-the-source
